src/github_projects_burndown_chart/chart/burndown.py

```diff
@@ -80,8 +80,12 @@ def __prepare_chart(self):
 
     def generate_chart(self, path):
         self.__prepare_chart()
-        if not os.path.exists(path):
-            os.makedirs(os.path.dirname(path))
+
+        # Create directories if it needs
+        dirname = os.path.dirname(path)
+        if not os.path.exists(dirname):
+            os.makedirs(dirname)
```

Comment:
```suggestion
        os.makedirs(dirname, exist_ok=True)
```

See https://docs.python.org/3/library/os.html#os.makedirs

Reply:
I made this change myself in ccefccb6523b4bbc09a6ed42449e0a3a1adbf214